compiler/ballerina-parser/src/main/resources/syntax_diagnostic_message.properties

```diff
@@ -277,3 +277,5 @@ error.invalid.whitespace.in.slash.lt.token=invalid whitespace in ''\\<''
 error.local.type.definition.not.allowed=local type definition not allowed
 error.type.inclusion.in.object.constructor=type inclusions are not allowed in object constructor
 error.version.in.import.declaration.no.longer.supported=version in import declaration no longer supported
+error.resource.path.rest.param.followed.by.resource.path.segment=\
+  resource path rest parameter followed by resource path segment
```

Comment:
how about something like
`rest parameter must be the last segment of resource path`

Reply:
+1